src/ResourceManager/Storage/Commands.Management.Storage.Test/ScenarioTests/StorageAccountTests.ps1

```diff
@@ -247,7 +250,7 @@ function Test-SetAzureStorageAccount
 
         $stotype = 'Standard_LRS';
         # TODO: Still need to do retry for Set-, even after Get- returns it.
-        Retry-IfException { Set-AzureRmStorageAccount -ResourceGroupName $rgname -Name $stoname -Type $stotype -DisableEncryptionService $encryptionServiceBF -EnableHttpsTrafficOnly $false; }
+        Retry-IfException { Set-AzureRmStorageAccount -ResourceGroupName $rgname -Name $stoname -Type $stotype -DisableEncryptionService $encryptionServiceBF -EnableHttpsTrafficOnly $false }
```

Comment:
For all set-AzureRMstorageAccount with -DisableEncryptionService/-EnableEncryptionService, should add "-StorageEncryption" if not add already.

Reply:
@blueww The tests were failing when I added -StorageEncryption (see my comment on #5243). Currently, the recordings created in the PR run without it and succeed.